usmqe/web/tendrl/mainpage/hosts/pages.py

```diff
@@ -0,0 +1,38 @@
+"""
+Hosts page abstraction.
+"""
+
+
+from webstr.core import WebstrPage
+from webstr.patternfly.contentviews import pages as contentviews
+
+import usmqe.web.tendrl.mainpage.hosts.models as m_hosts
+
+
+class HostsMenu(WebstrPage):
+    """
+    page object for hosts top menu
+    """
+    _model = m_hosts.HostsMenuModel
+    _label = 'hosts top menu'
+    _required_elems = ['header', 'search']
+
+# TODO use search
```

Comment:
Well, it's named as filter now, once again copy-paste code. I could change it to use filter though :-)